Gemfile.lock

```diff
@@ -1,60 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.0)
+    better_errors (2.5.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (10.0.2)
+    byebug (11.0.1)
     coderay (1.1.2)
     commander (4.4.7)
       highline (~> 2.0.0)
     database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubi (1.7.1)
-    event_sourcery (0.22.0)
+    erubi (1.9.0)
+    event_sourcery (0.23.0)
     event_sourcery-postgres (0.8.0)
       event_sourcery (>= 0.14.0)
       pg
       sequel (>= 4.38)
-    highline (2.0.0)
+    highline (2.0.3)
```

Comment:
### [highline](https://github.com/JEG2/highline)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.0 → 2.0.3

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **17** commits. See the full changes on [the compare page](https://github.com/JEG2/highline/compare/v2.0.0...v2.0.3).

These are the first 10 commits:
- (104be00) [Quick fix for #237](https://github.com/JEG2/highline/commit/104be004c62deeb34872f490df253bbba18ed3f4)
- (a64b523) [Apply Rubocop suggestions](https://github.com/JEG2/highline/commit/a64b52365b7ebb1f8a16999d518b688182c22292)
- (3896c9c) [Bump up version to 2.0.1](https://github.com/JEG2/highline/commit/3896c9c37b9fa34be745e1bc1a66bb5eeffc998f)
- (ff972b5) [Merge pull request #238 from JEG2/issue_237](https://github.com/JEG2/highline/commit/ff972b56807cf2303404ec8b0de7084b5f9a8fe5)
- (3729b17) [Avoid YARD warning UnknownParam](https://github.com/JEG2/highline/commit/3729b17fa6f2ec880ee0fd8f402a2def14e5d4c2)
- (fa8fb50) [Merge pull request #240 from olleolleolle/patch-1](https://github.com/JEG2/highline/commit/fa8fb5055d6f9d1f10fa4e3765bcb355b359fc16)
- (c4796b4) [CI: Add 2.6](https://github.com/JEG2/highline/commit/c4796b4977566ad9b990d240c023093131760ce9)
- (defb6ec) [Merge pull request #241 from olleolleolle/patch-1](https://github.com/JEG2/highline/commit/defb6ec8d93448b79a4f20cd3d0c49da80e8df1f)
- (69c8ea5) [Add new capital_letter option to menu index](https://github.com/JEG2/highline/commit/69c8ea5b017634a1ae92141eab38967dc6a395b6)
- (154754b) [Refactor letter options for index](https://github.com/JEG2/highline/commit/154754be7399229351c2135a3051a0bc11ece4fb)


</details>